src/components/App.js

```diff
@@ -15,7 +15,7 @@ class App extends React.Component {
   render() {
     const { dummy } = this.state;
     return (
-      <div>
+      <div className="app d-flex">
         {dummy}
```

Comment:
What is the purpose of dummy? :thinking:

Reply:
Hi @Oghenebrume50,
https://github.com/fivan18/calculator-react/blob/778fcad3ec938372b212039992ccaf9c4bdb7fdc/src/components/App.js#L9-L14

I'll refactor it at the final milestone, thanks for the review.